app/server/src/routes/enrollment/enrollment.validate.js

```diff
@@ -5,10 +5,29 @@ exports.validate = (method) => {
     case "createEnrollment": {
       return [
         body("user_id", "user_id does not exist").exists(),
+        body("course", "course does not exist").exists(),
         body("course_id", "course_id does not exist").exists(),
+        body("is_active", "is_active does not exist").exists(),
+        body("notes", "notes does not exist").exists(),
+        body("progress", "progress does not exist").exists(),
       ];
     }
     case "getEnrolledCourses": {
+      return [
```

Comment:
similarly, if you have not tested this, you might want. It might give errors. I would suggest we remove cases for get requests.

Reply:
removed case!